OPHD/Population/Population.cpp

```diff
@@ -98,156 +99,146 @@ int Population::adults() const
 void Population::spawn_children(int morale, int residences, int nurseries)
 {
 	if (residences < 1 && nurseries < 1) { return; }
+	if (mPopulation[PersonRole::ROLE_SCIENTIST] + mPopulation[PersonRole::ROLE_WORKER] < 1) { return; }
 
-	// This should be adjusted to maybe two or three kids per couple to allow for higher growth rates.
-	if (mPopulation[PersonRole::ROLE_SCIENTIST] + mPopulation[PersonRole::ROLE_WORKER] > mPopulation[PersonRole::ROLE_CHILD])
```

Comment:
It seems this check was modified, rather than simply inverted to reduce nesting. Not sure if that was intentional or not. The commit comment would suggest the change wasn't intentional, though the comment being removed here maybe suggests perhaps it was?

Reply:
It was intentional. I noticed that it was actually looking at the wrong values so I took the opportunity to fix it. Should have made it a separate commit.

Reply:
Not too sure what you mean by looking at the wrong values.

I believe the original code roughly matched how Outpost 2 worked. You couldn't raise new kids if they already exceeded the number of adults. I guess only so much time to work, look after kids, and make kids? Of course, those kids will soon turn into adults, and that happens faster than adults die, so population growth is of course still quite possible.